demes/demes.py

```diff
@@ -1877,7 +1870,7 @@ def discrete_demographic_events(self) -> Dict[str, List[Any]]:
             )
         return demo_events
 
-    def in_generations(self):
+    def in_generations(self) -> "Graph":
```

Comment:
Maybe use the ``from __future__`` thing so you can write it without the quotes? Or is that only available in 3.7+?

Reply:
Unfortunately that was introduced in 3.7. BTW, that future becomes the default behaviour in 3.10.